src/gluonts/model/seq2seq/_mq_dnn_estimator.py

```diff
@@ -237,11 +240,14 @@ def __init__(
             prefix="decoder_",
         )
 
-        quantile_output = (
-            QuantileOutput(self.quantiles, is_iqf=self.is_iqf)
-            if self.quantiles
-            else None
-        )
+        if self.quantiles:
+            quantile_output = (
+                IncrementalQuantileOutput(self.quantiles)
+                if is_iqf
```

Comment:
Not a fan of this inline `if` here, can't we do

```py
if not self.quantiles:
    quantile_output = None
elif is_iqf:
    quantile_output = IncrementalQuantileOutput(self.quantiles)
else:
    quantile_output = QuantileOutput(self.quantiles)
```

Reply:
Sure, why not